lua/wire/server/wirelib.lua

```diff
@@ -1276,6 +1276,28 @@ function WireLib.IsValidMaterial(material)
 	return material
 end
 
+local escapes = { n = "\n", r = "\r", t = "\t", ["\\"] = "\\", ["'"] = "'", ["\""] = "\"" }
+--- Parses a user-generated string so escape characters become their intended targets. Note that this is not a complete implementation of escape characters.
+--- @param str string
+function WireLib.ParseString(str)
+	return (str:gsub("\\([%a\\]?)(%d?%d?%d?)", function(char, num)
```

Comment:
The `[%a\\]` pattern won't work for ' or ". Also shouldn't the num check also be `elseif num~="" then`?

Reply:
If my second question is true, then you can also get rid of the `or ""` for each of the num concats.

Reply:
First is an easy fix but second one, in my testing num can only be nil or a valid string. Either way irrelevant now.